GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -194,7 +194,7 @@ void Renderer::renderCamera(Scene* scene, Camera* camera, int framebufferId,
 
     bool renderShadow = cameraLight; // TODO reader from scene;
 
-    if ((framebufferId != 0) && !renderShadow) {
+    if (!renderShadow) {
```

Comment:
There was an issue with the monoscopic renderer and the rendering with the shadow map itself when we first did this a while back, but, i guess it can be removed now, since the crash is a critical issue.
I'll work on a adaptation of the shadow map with this renderer.